src/script/standard.cpp

```diff
@@ -317,3 +317,7 @@ CScript GetScriptForWitness(const CScript& redeemscript)
     ret << OP_0 << ToByteVector(hash);
     return ret;
 }
+
+bool IsValidDestination(const CTxDestination& dest) {
+    return dest.which() != 0;
```

Comment:
Maybe add a quick comment that this is CNoDestination so it comes up when grepping for future changes.

Reply:
Could use `!(operator==)` instead?